src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -774,7 +781,10 @@ static std::string parseAttributeList(ConnectionInfo* ciP, std::vector<std::stri
     {
       return badInput(ciP, "attrs element has forbidden char");
     }
-
+    if (attrName.length() > MAX_ID_LEN)
```

Comment:
If MAX_ID_LEN is to be used for more than 'id', perhaps we should give it a more generic name?
Like MAX_FIELD_LEN or something like that ...